deploy/examples/object.yaml

```
@@ -116,6 +116,7 @@ spec:
       disabled: false
     readinessProbe:
       disabled: false
+  # rgwOpsLogEnabled: true
```

It's simpler to read the example if we uncomment and set the default value
```suggestion
  rgwOpsLogEnabled: false
```